### ISSUE TITLE
Action icon overlays unusable on dark images

### ISSUE THREAD

**Issue by i-a-n** [reactioncommerce/reaction#386]
Product images with a dark background are practically unusable, as the iconography overlays are too hard to see. Icons appear to be CSS psuedo-elements of an icon font, so something as simple as a light text-shadow should fix this. Note an example with a background color of #323232:

![example](http://union.io/images/repo/20150612-00--1797c6.png)

**Comment by mikemurray** [comment]
Might be better to wrap the buttons in a container of some sort. See example:

![artboard 1](https://cloud.githubusercontent.com/assets/42217/8122989/233d4532-1077-11e5-8d0d-1d12b83b3a4f.png)

**Comment by i-a-n** [comment]
Nice. That one gets my vote!

**Comment by aaronjudd** [comment]
@mikemurray :+1:  gets my vote.  nice thing about the center controls  - no complications when using RTL.

**Comment by mikemurray** [comment]
Changes are under the **feature-controls-386** branches in [reaction-core](https://github.com/reactioncommerce/reaction-core/tree/feature-controls-386) and [reaction-core-theme](https://github.com/reactioncommerce/reaction-core-theme/tree/feature-controls-386).

Also replaced the clone icon with one that looks like duplicating files.

Preview (LTR):

![screen shot 2015-06-16 at 9 39 34 pm](https://cloud.githubusercontent.com/assets/42217/8199887/ba505a6a-1471-11e5-9eef-99263ad9750b.png)

And RTL:
![screen shot 2015-06-16 at 9 59 18 pm](https://cloud.githubusercontent.com/assets/42217/8199980/ff7fb206-1472-11e5-80e6-3610a58982b2.png)

**Comment by aaronjudd** [comment]
I like it.  Does seem a touch large when I'm seeing it on my screen. However,  it looks great in the screenshots.  I think I'm just not used to seeing it yet. It's certainly a big improvement though, we should merge it.

![screenshot 2015-06-17 10 43 56](https://cloud.githubusercontent.com/assets/439959/8214411/e6089fde-14dd-11e5-9791-f2116c450564.png)

I noticed my onRendered update the other day broke drag n drop, and popovers.  I've committed an update to your branch that resolves this.
